client/src/partials/reports_proposed/debtor_group/balance.js

```diff
@@ -0,0 +1,107 @@
+angular.module('bhima.controllers')
+.controller('configureBalance', [
+  '$scope',
+  '$http',
+  '$sce',
+  'validate',
+  function ($scope, $http, $sce, validate) {
+
+    var dependencies = {};
+
+    // Configuration objects optionally passed to /report/build - drives configuration UI
```

Comment:
Curiously, I have no memory of writing any of the debtor_group balance code.  I'll delete this file and all the other balance duplicates.  No idea how that happened, possibly a broken merge.